setup.cfg

```diff
@@ -2,8 +2,12 @@
 name = python-typing-update
 author = Marc Mueller
 description = Update Python typing syntax
+long_description = file:README.md
+long_description_content_type = text/markdown
 keywords = typing,pep585,pep604
 license = MIT
+license_file = LICENSE
```

Comment:
The license file is already included in the wheel.
https://wheel.readthedocs.io/en/stable/user_guide.html#including-license-files-in-the-generated-wheel-file

I'm not sure about the source distribution though. Might be that it's missing there. I'll probably add a `MANIFEST.in` file to include it there too.

Reply:
Small update: `setuptools` only includes license files if they are mentioned in `license_file`, `license_files` or `MANIFEST.in`.